css/style.css

```diff
@@ -0,0 +1,599 @@
+  font-size: 13px;
+  color: rgba(39, 39, 39, .5);
+  ;
+}
+
+#culture-n-travel div article p:last-of-type {
+  font-size: 13px;
+  color: black;
+  background-color: #c2d4d7;
+  display: inline-block;
+  padding-right: 3px;
+  padding-left: 3px;
+}
+
+#culture-n-travel div article p:last-of-type:hover {
+  color: #c2d4d7;
+  background-color: black;
+}
+
+#more-stories {
+  margin-top: 20px;
+}
+
+#more-stories>article {
+  padding-top: 20px;
+}
+
+#more-stories>article>div>img {
+  width: 135px;
+  display: block;
+}
+
+#more-stories>article>div:last-of-type {
+  padding-left: 5px;
+}
+
+#more-stories>article>div:last-of-type h4 {
+  font-weight: bold;
+  font-family: "Roboto Condensed", sans-serif;
+  font-size: 17px;
+}
+
+#more-stories>article>div:last-of-type p {
+  font-size: 15px;
+}
+
+#opinion article div:last-of-type {
+  padding-left: 15px;
+}
+
+#opinion article div:last-of-type p:first-of-type {
+  font-weight: bold;
+  font-family: "Roboto Condensed", sans-serif;
+  font-size: 17px;
+  color: black;
+  line-height: 20px;
+  margin-bottom: 5px;
+}
+
+#opinion article div:last-of-type p:last-of-type {
+  color: #666;
+  font-weight: bold;
+  margin-bottom: 5px;
+}
+
+#opinion article {
+  margin-top: 10px;
+}
+
+#opinion {
+  margin-bottom: 5px;
+}
+
+#last-news article {
+  margin-top: 10px;
+  padding-top: 10px;
+}
+
+#last-news article div img {
+  width: 139px;
+}
+
+#last-news article p {
+  padding-left: 15px;
+  font-weight: bold;
+  font-family: "Roboto Condensed", sans-serif;
+  font-size: 15px;
+}
+
+#newsletter-singup {
+  margin-top: 15px;
+  background-color: #f4f5f6;
+  border: 1px solid #e5e5e5;
+  padding: 20px;
+  max-width: 300px;
+  margin: auto;
+  margin-top: 10px;
+}
+
+#newsletter-singup div:first-of-type div {
+  background-color: #f72210;
+  height: 30px;
+  width: 30px;
+}
+
+#newsletter-singup div:first-of-type div img {
+  height: 100%;
+  width: 100%;
+}
+
+#newsletter-singup div:first-of-type h3 {
+  font-size: 19px;
+  line-height: 15px;
+  font-weight: bold;
+  padding-left: 5px;
+  margin-bottom: 0px;
+  text-transform: uppercase;
+}
+
+#newsletter-singup button {
+  margin-top: 15px;
+  border: 0;
+  background-color: #666666;
+  color: white;
+  padding: 10px;
+  font-weight: bold;
+  margin-bottom: 5px;
+}
+
+#newsletter-singup>p {
+  font-weight: bold;
+  font-family: "Roboto Condensed", sans-serif;
+  font-size: 12px;
+}
+
+#part-2 {
+  margin-top: 10px;
+}
+
+#part-2 img {
+  width: 100%;
+  height: 290px;
+}
+
+#part-2>div:first-child>article:first-child img {
+  height: 387px;
+}
+
+#part-2 article>div:first-of-type {
+  position: relative;
+}
+
+#part-2 article>div:first-of-type p {
+  position: absolute;
+  bottom: 0;
+  background: white;
+  color: red;
+  margin-bottom: 0;
+  width: 100%;
+  padding-top: 10px;
+}
+
+#part-2 article>div:first-of-type p span {
+  background: red;
+  color: white;
+  margin: 0;
+  padding-right: 5px;
+  display: inline-block;
```

Comment:
`SUGGESTION`

You can also use the `d-inline-block` class name for this selector element and in all other occurrences.